messaging/messaging.go

```diff
@@ -225,6 +224,7 @@ type WebpushConfig struct {
 	Headers      map[string]string    `json:"headers,omitempty"`
 	Data         map[string]string    `json:"data,omitempty"`
 	Notification *WebpushNotification `json:"notification,omitempty"`
+	FcmOptions   *WebpushFcmOptions   `json:"fcm_options,omitempty"`
```

Comment:
FCM team tells me that this json key should be called `fcmOptions`. `fcm_options` is a documentation error, and they are working on it.

Reply:
OK :) fixed